### PR TITLE
new PR for wall avoidance

### PR DESCRIPTION
**Pull request by tomsilver**
@chetan51 step 1...

### CODE REVIEW

**Comment by chetan51** [comment]
I think you forgot to include the modifications to the car prefab!

**Comment by tomsilver** [comment]
added the car prefab

File: vehicle-control/simulation/Assets/Standard Assets/Vehicles/Car/Prefabs/Car.prefab [hunk 1]
```diff
@@ -535,6 +535,24 @@ GameObject:
   - 4: {fileID: 466396}
   - 114: {fileID: 11458312}
   - 120: {fileID: 12050378}
+  - 114: {fileID: 11465580}
+  m_Layer: 0
+  m_Name: DepthCollectionSensor
```
Comment (chetan51) [inline vehicle-control/simulation/Assets/Standard Assets/Vehicles/Car/Prefabs/Car.prefab]: I think this change belongs in the other PR (https://github.com/numenta/nupic.research/pull/191/files)

**Comment by tomsilver** [comment]
opening a new PR for this

**Pull request closed without merging**